colcon_ros_cargo/package_identification/ament_cargo.py

```diff
@@ -43,6 +45,14 @@ def identify(self, metadata):  # noqa: D102
                 'Got build type ament_cargo but could not find "Cargo.toml"')
             return
 
+        ament_build = 'cargo ament-build --help'.split()
+        if subprocess.run(ament_build, capture_output=True).returncode != 0:
+            logger.error(
+                'ament_cargo package found but cargo ament-build was not '
+                'detected. Please install it by running: '
+                '`cargo install cargo-ament-build`')
```

Comment:
This is a super minor nitpick but I wonder if we should tweak this line to
```
'\n $ cargo install cargo-ament-build'
```
Putting the solution on its own line and beginning it with a `$ ` may users noticed the command and immediately clue them in that it's a recommendation to run a command.

I might also be a little inclined to `\n` at the start of `Please install it by running:` but that's less important.

Reply:
Tried that out, before (without the newline):

![image](https://github.com/user-attachments/assets/3acfd502-acfe-42a9-bccc-c9f3ddbb7396)

After (with the newline):

![image](https://github.com/user-attachments/assets/5e45ca00-fe71-4358-8220-671d9d3de465)

I have to say I don't actually like the newline approach, colcon logging adds a header to the line specifyin where / when the error originated, the moment we add a newline that header is lost and the output might be confused with the rest of the command's output.
A probably better option is to just add a new `logger` call so the header is included, but this also doesn't look terribly clean:

![image](https://github.com/user-attachments/assets/9d206198-3865-4be8-a647-2d4cf34a8315)

What do you think?

Reply:
The header really does add a lot of noise and makes the warning easy to overlook.

Here's a PR based on my own experimental iteration on how to display the warning: https://github.com/luca-della-vedova/colcon-ros-cargo/pull/4